dart/lib/src/default_integrations.dart

```diff
@@ -7,16 +7,18 @@ import 'sentry.dart';
 import 'sentry_options.dart';
 import 'throwable_mechanism.dart';
 
-/// integration that capture errors on the runZonedGuarded error handler
+/// Integration that runs runner function within runZonedGuarded
 class RunZonedGuardedIntegration extends Integration {
-  final AppRunner _appRunner;
+  RunZonedGuardedIntegration();
 
-  RunZonedGuardedIntegration(this._appRunner);
+  FutureOr<void> Function() runner;
```

Comment:
is there a way to go around this public getter/setter? via ctor or something else?